code/modules/admin/admin_ranks.dm

```diff
@@ -120,7 +120,7 @@ var/list/forum_groupids_to_ranks = list()
 			load_admins()
 			return
 
-		var/DBQuery/query = dbcon.NewQuery("SELECT ckey, rank, flags FROM ss13_admins;")
+		var/DBQuery/query = dbcon.NewQuery("SELECT ckey, `rank`, flags FROM ss13_admins;")
```

Comment:
That change confuses me.
Why is only rank quoted, but the rest of the columns not?

Reply:
Because "rank" is a keyword in the latest versions of MySQL.

See
https://www.geeksforgeeks.org/mysql-ranking-functions/